src/modules/sync/hooks/use-sync-dialog-texts.tsx

```diff
@@ -36,19 +36,19 @@ export const useSyncDialogTexts = ( type: 'pull' | 'push' ) => {
 				staging: {
 					title: __( 'Push to Staging' ),
 					description: __(
-						'Pushing will replace the existing files and database with a copy from your local site.\n\n The staging site will be backed up before any changes are applied.'
+						"Pushing will overwrite your staging site's selected files and database with content from your local site. You can push everything or select specific parts. Unchecked items will not be changed. The staging site will be backed up before any changes are applied."
 					),
 				},
 				sandbox: {
 					title: __( 'Push to Sandbox' ),
 					description: __(
-						'Pushing will replace the existing files and database with a copy from your local site.\n\n The sandbox site will be backed up before any changes are applied.'
+						"Pushing will overwrite your sandbox site's selected files and database with content from your local site. You can push everything or select specific parts. Unchecked items will not be changed. The sandbox site will be backed up before any changes are applied."
```

Comment:
```suggestion
						"Pushing will overwrite your sandbox site's selected files and database with content from your local site. Unchecked items will not be changed. The sandbox site will be backed up before any changes are applied."
```

Reply:
Totally agree 👍 But we could kinda merge two sentences with adding `/or` to avoid extra sentence and to be direct that one or the other or both can be selected.

```suggestion
						"Pushing will overwrite your sandbox site's selected files and/or database with content from your local site. Unchecked items will not be changed. The sandbox site will be backed up before any changes are applied."
```

Reply:
Thanks for the suggestion, @sejas. It makes sense to me. I updated the description in [a509562](https://github.com/Automattic/studio/pull/1536/commits/a50956268552f1d97debde886ee3cfd2ecc1b160).

Reply:
@sejas I don't have any better suggestions, so I vote we go with removing `You can push everything or select specific parts`. We can always tweak in the future.

Reply:
Great. @ndiego Thanks for your input.